### PR TITLE
Remove activationCode from sensor entities

### PR DESCRIPTION
**Pull request by aiwilliams**

### CODE REVIEW

File: src/converters.ts [hunk 1]
```diff
@@ -134,6 +134,7 @@ export function createDeviceSensorEntity(
         function: ["anti-malware", "activity-monitor"],
         macAddress: formatMacAddress(data.mac_address),
         lastSeenOn: getTime(data.last_contact_time),
+        activationCode: null, // do not store what is essentially a secret
```
Comment (austinkelleher) [inline src/converters.ts:137]: Should we remove this from the raw data too?

Comment (ndowmon) [reply src/converters.ts:137]: It would be great to add a single recording test to this for review/validation. If it's not too much trouble.

**Review by mknoedel — APPROVED** [review]
I don't think it's important to rerecord tests on a change this small so long as the tests still pass

**Comment by ndowmon** [comment]
>I don't think it's important to rerecord tests on a change this small so long as the tests still pass

As a reviewer, it would just make me feel confident that we haven't accidentally left sensitive data exposed in `_rawData`. I'm a glutton for proof.

**Comment by ndowmon** [comment]
>I don't think it's important to rerecord tests on a change this small so long as the tests still pass

As a reviewer, it would just make me feel confident that we haven't accidentally left sensitive data exposed in `_rawData`. I'm a glutton for proof.

**Review by austinkelleher — DISMISSED** [review]

File: src/converters.ts (hunk 1 above)
Comment (aiwilliams) [reply src/converters.ts]: Great catch, removed the value from `_rawData` as well, and made both have `REDACTED` value to communicated that we saw a value and removed it.

**Review by ndowmon — APPROVED** [review]

**Pull request merged**